.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: Please do NOT file bugs without filling in this form.
+title: '[Bug] '
+labels: ''
+assignees: ''
+
+---
+
+**Which version of Microsoft.IdentityModel are you using?**
+Note that to get help, you need to run the latest version. 
+<!-- E.g. Microsoft.IdentityModel 6.14 -->
+
+**Where is the issue?**
+ * [ ] M.IM.JsonWebTokens
+ * [ ] M.IM.Protocols.OpenIdConnect
+ * [ ] M.IM.Tokens
+ * [ ] M.IM.Tokens.Jwt
+ * [ ] M.IM.Validators
+* Other (please describe)
+
+**Is this a new or an existing app?**
+<!-- Ex:
+a. The app is in production and I have upgraded to a new version of Microsoft.IdentityModel.*
+b. The app is in production and I haven't upgraded Microsoft.IdentityModel.*, but started seeing this issue.
+c. This is a new app or an experiment.
+-->
+
+**Repro**
+
+```csharp
+var your = (code) => here;
+```
+
+**Expected behavior**
+A clear and concise description of what you expected to happen (or code).
+
+**Actual behavior**
+A clear and concise description of what happens, e.g. an exception is thrown, UI freezes.
+
+**Possible solution**
+<!--- Only if you have suggestions on a fix for the bug. -->
+
+**Additional context / logs / screenshots / links to code**
+Add any other context about the problem here, such as logs and screenshots or links to code.
```

Comment:
nit: Might be worth reminding folks not to include an PII (as we do in ICM templates etc)

Reply:
good point. will add in the other repos as well.